server/src/main/java/org/opensearch/index/mapper/MapperService.java

```diff
@@ -147,8 +147,8 @@ public enum MergeReason {
     );
     public static final Setting<Long> INDEX_MAPPING_DEPTH_LIMIT_SETTING = Setting.longSetting(
         "index.mapping.depth.limit",
-        20L,
-        1,
+        100L,
```

Comment:
This change seems unrelated, what is the reason?

Reply:
Thick finger. undoing.